docs/design/Liqo-Agent.md

```diff
@@ -0,0 +1,103 @@
+# Liqo Agent
+## Overview
+**Liqo Agent** is the Liqo component that lets the user interact with Liqo in an easy, friendly way.
+It represents one of the two main entry point to the Liqo environment, along with the **LiqoDash**.
+
+_Agent_ is accessible from its tray icon and provides a menu trough which users can:
+
+* monitor the **status** of their own Liqo
+* get **quick access** to Liqo **main features**, e.g. start/stop, Liqo working mode,
+peering management
+* manage the Agent settings, e.g. notifications
+
+_Agent_ aims at enhancing the Liqo users' experience, especially for non-pro ones, 
+allowing them to perform simple operations without needing to interact with the **LiqoDash**.
+Moreover, thanks to the notification system, one gets quickly informed about the main events, such as
+peering's status or the arrival of new Advertisements.
+
+### Features
+*   **Status information**: get details about Liqo:
+    * Liqo turned on/off
+    * Liqo work mode 
+    * active peerings
+*   **Start/Stop** Liqo
+*   **Choose** Liqo **work mode**:
+    * **_Autonomous_**: The device uses its own on-board intelligence (local orchestrator). 
+    This way it can:
+        * work using **only its own resources**
+        * connect to multiple peers, both **consuming** (under its control) **foreign resources**
+        and **sharing its proprietary resources** to other peers. 
+        The system acts as two cooperating nodes.
+        Each sharing operation is independent of the others.    
+    * **_Tethered_**: The device connects to a foreign _Autonomous_ Liqo peer (e.g. the corporate network) 
+    and lets the foreign orchestrator **remotely control the usage of its resources**
+        * When the tethering stops (e.g. the device leaves the foreign network), the local on-board
+        intelligence takes back the control of the home cluster. 
+        * In this mode, Liqo can have **only one active peer**.    
+*   Get quick **access** to **LiqoDash**
+*   Manage Liqo Agent **settings**
+    * Users can turn on/off notifications and choose how they would like to receive them:
+        * silent notifications on the tray icon
+        * tray icon in combination with desktop banners 
+*   **Peerings** and **Advertisements management**: using a '_networks management -like_' interface,
+users can monitor available Liqo peers, read the received Advertisements (offers) 
+and manage the connections.
+
+### Limitations
+
+In order to orchestrate the tray icon and menu, _Agent_ exploits the [systray](https://github.com/getlantern/systray)
+package which has some limitations:
+* Missing sub-menu implementation. _Agent_ overcame the problem using native OS graphic windows.
+* The tray menu works as a stack, and the library offers no possibility to delete an Item. 
```

Comment:
Those are limitations from an implementation point of view. I feel that this section should be more oriented about what user that install the agent today cannot perform (e.g.; start/stop liqo).